common/src/buffer.c

```diff
@@ -0,0 +1,667 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
>Licensed [](start = 3, length = 8)

I still think we can remove buffer from this repo, we only need constbuffer

Reply:
removed buffer in next iteration.

---
In reply to: [451918485](https://github.com/Azure/azure-c-pal/pull/7#discussion_r451918485) [](ancestors = 451918485)